frontend/packages/console-shared/src/components/status/Icons.tsx

```diff
@@ -9,21 +9,19 @@ import {
   global_danger_color_100 as dangerColor,
 } from '@patternfly/react-tokens';
 
-export type ColoredIconProps = {
+type ColoredIconProps = {
   className?: string;
   alt?: string;
 };
 
-export const GreenCheckCircleIcon: React.FC<ColoredIconProps> = ({ className, alt }) => {
-  return (
-    <CheckCircleIcon color="var(--pf-chart-color-green-400)" className={className} alt={alt} />
-  );
-};
+export const GreenCheckCircleIcon: React.FC<ColoredIconProps> = ({ className, alt }) => (
+  <CheckCircleIcon color="var(--pf-chart-color-green-400)" className={className} alt={alt} />
```

Comment:
Not related to this change but it would be good to use the variable from react-tokens instead of hardcoding it. (See dangerColor and warningColor)

Reply:
done